### ISSUE TITLE
Class 'PHPJasper\PHPJasper' not found 

### ISSUE THREAD

**Issue by Karacraft** [PHPJasper/phpjasper#137]
**Describe the bug**
```
Symfony \ Component \ Debug \ Exception \ FatalThrowableError (E_ERROR)
Class 'PhpJasper\PhpJasper' not found
```

I just installed PHPjasper, and followed all steps (with exception of compile, since i have JesperStudio).

 - OS: [Windows 10 1803]
 - Browser [Firefox]
 - Version [e.g. 65.0.22 64bit]

The Code is in web.php
```
$input = public_path() . '/report/hello_world.jasper';
    $output = public_path() . '/report/'.time().'_hello_world';
    $options = [
        'format' => ['pdf', 'rtf'] 
    ];
    $report = new PhpJasper;
    
    $report->process(
        $input,
        $output,
        $options
    )->execute();
```

**Comment by Karacraft** [comment]
It seems, i was looking at a very old post by geek.com from  [Geek.com](https://devhub.io/repos/geekcom-phpjasper) (since it's for Laravel).
I've found out after several attempts, that the class name is 
```
PHPJasper
````
So, I corrected all the mistakes and i can confirm that `"geekcom/phpjasper": "^2.7"` is working in perfect harmony with `Laravel 5.8`.

I request @geekcom  to update his page and add up a better readme for Laravel users as well.
**closing the issue**
